### ISSUE TITLE
Implement the service schedules for family planning follow-up visits

### ISSUE THREAD

**Issue by msschroeder** [OpenSRP/opensrp-client-chw-hf#40]
- [ ] Service schedule will only occur for those where: {fp_method_accepted} = "POP", "COC", "Female condom", "Male condom", "Injectable", "female sterilization", "IUCD"
- [ ] A recurring visit will occur for those where: {fp_method_accepted} = "POP", "COC", "Female condom", "Male condom", "Injectable"
- [ ] Each type fo family planning (pills - POP, COC, condoms - male, female, and injectable) will have their own schedule
- [ ] POP/COC refill will be dependant on the {fp_start_cocpop} and {no_pillcycles} provided in the family planning registration for the first schedule and then {fp_visit_date} and {no_pillcycles} filled in the follow-up visits.
- [ ] The condom will have a schedule based on a monthly schedule
- [ ] The injectable will be dependant on the {fp_start_injectable} in the family planning registration and then {fp_refill_injectable} from the follow-up visit
- [ ] A non-recurring visit will occur for those where: {fp_method_accepted} = "female sterilization", "IUCD" 
- [ ] Each type fo family planning (sterilization, IUCD ) will have their own schedule
- [ ] The IUCD will have two visits (IUCD follow up 1, IUCD follow up 2) defined in the data dictionary
- [ ] The Sterilization will have three visits (Female sterilization follow-up 1, Female sterilization follow-up 2, Female sterilization follow-up 3) defined in the data dictionary
- [data dictionary]
(https://docs.google.com/spreadsheets/d/1xJtrHmKqEVuWjvzAcewEavATrfkw9mN-Okf_zGR544w/edit?usp=sharing) - Service Schedule - CHW Tab

**Comment by allan-on** [comment]
Hi @msschroeder. Just to clarify, are these service schedules for determining the status of the follow-up visit button in https://github.com/OpenSRP/opensrp-client-chw-hf/issues/38 
This is because in the Family Planning Register in https://github.com/OpenSRP/opensrp-client-chw-hf/issues/34 there are no services due/overdue buttons showing :thinking:

**Comment by msschroeder** [comment]
@allan-allay Yes, these are for determining the follow up service schedules for the follow up visits.
I think we should have a chat tomorrow first thing

**Comment by allan-on** [comment]
Hi @msschroeder. The service schedules have already been implemented but nature of the bug with these isn't documented and so I was wondering what needs to be looked into on this issue.

**Comment by msschroeder** [comment]
@allan-allay It just wasn't clear how the follow-up visit was working form and this was related. So there may not be an issue. I will move this to QA then.

**Comment by msschroeder** [comment]
@allan-allay @Mstjamush So my thoughts with this issue and testing were the following:

1. I can either create an a user on HF or CHW and sync back
2. I fill in the FP initiation form. 
3. Sync both the HF and CHW
4. Check to see if the FP was successful on the CHW side. 
5. Check the schedules on the FP register

I got to step 4. It seems that information is not syncing back to the CHW from the HF - even in the creation of users. So I was unable to check the schedules (#92 )
